go/s3/UploadDirectory/dummy/dummy.txt

```diff
@@ -0,0 +1 @@
+What a dummy.
```

Comment:
Can you change to "This is dummy text."?

Reply:
Done.